install/setup.sh

```diff
@@ -6,6 +6,9 @@ source variables.sh
 # Set up the NFS server on the Head node, and mount the NFS server on each of the clients
 source setup_nfs.sh
 
+# Check if the shared dir collaboration is working properly or not
+source shared_disk_check.sh
```

Comment:
If this fails, does anything happen or does the script keep going? May want to stop the script or make it very clear that the final deployment will not work as intended

Reply:
true, I will change to stop the execution